spec/template.yml

```diff
@@ -0,0 +1,183 @@
+apiVersion: v1
```

Comment:
```spec/vcr_cassettes/manageiq/providers/openshift/container_manager/``` is a good start. Give it a descriptive name so we know this template is used to create templates.